### PR TITLE
Added a few entries.

### PR DESCRIPTION
**Pull request by juliendelplanque**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +100,11 @@ A categorized community-driven collection of awesome Pharo libraries, tools, fra
 + [Mocketry](https://github.com/dionisiydk/Mocketry) - Mock objects library with very fluent lightweight API.
 + [StateSpecs](https://github.com/dionisiydk/StateSpecs) - Assertions library based on should expressions.
 
+## Testing
++ [Babymock](https://github.com/zeroflag/BabyMock) - A visual mock object library.
++ [Mocketry](https://github.com/dionisiydk/Mocketry) - Mock objects framework.
```
Comment (jecisc) [inline README.md]: Mocketry was added in a TDD section. 

Maybe TDD could be a subsection TDD to Testing?

Comment (juliendelplanque) [reply README.md]: done.

**Comment by juliendelplanque** [comment]
Is this PR ok now? @jecisc

**Comment by jecisc** [comment]
Also I would keep Denis description of Mocketry.

**Comment by juliendelplanque** [comment]
I think this PR is finally ok @jecisc

**Pull request merged**